base/_base.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
```

Comment:
I'm on board with creating roll ups for partials. In this case, _box-model makes sense here. I was struggling to figure out where to drop these in